packages/effect/src/Stream.ts

```diff
@@ -4558,10 +4558,10 @@ export const encodeText: <E, R>(self: Stream<string, E, R>) => Stream<Uint8Array
 
 /**
  * Creates a `Stream` using addEventListener.
- * @since 1.0.0
+ * @since 3.1.0
  */
-export const fromEventListener: (
+export const fromEventListener: <A extends Event = Event>(
```

Comment:
```suggestion
export const fromEventListener: <A = Event>(
```

Just in case someone wants to use something non-standard.